Application/src/view/visuals/component/Message.java

```diff
@@ -231,11 +237,27 @@ public void setStatic(boolean staticIndicator)  {
     public void renderDefault(GraphicsContext gc) {
 
 
+        if(trails.size() != 0) {
+            Trail t;
+            //Draws all trails except the first and last one
+            for (int i = 1; i < trails.size() - 1; i++) {
+                // Points to the first index of the trail array
+                t = trails.get(i);
```

Comment:
t is not really descriptive

Reply:
@timjon but @IsabelleTornqvist wants it this way

Reply:
Im blocking cause of this

Reply:
@IsabelleTornqvist wanted it to be exactly like her old code

Reply:
@timjon t should be fine because the comment above it explains that

Reply:
So here is the issue because of what was discussed above: if the variable name hadn't been changed from "t", then Tim couldn't question the name, because it wouldnt be apart of this pull request. But since it is changed, you are the author of that variable name, and in extent, "responsible" for it

Reply:
So the argument "Isabelle wanted it this way" isn't applicable